starboard/shared/ffmpeg/BUILD.gn

```diff
@@ -119,5 +119,6 @@ target(gtest_target_type, "ffmpeg_demuxer_test") {
     "//testing/gmock",
     "//testing/gtest",
   ]
-  data_deps = [ "//media/test/data:media_testdata" ]
+
+  # data_deps = [ "//media/test/data:media_testdata" ]
```

Comment:
@xiaomings is there an appropriate substitute for this now? (it doesn't exist in the chromium repo)